jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -511,7 +508,12 @@ public void exit(int status) {
 		Shutdown.exit(status);
 	}
 
-	/*[ENDIF] OPENJDK_METHODHANDLES*/
+	/*[IF JAVA_SPEC_VERSION >= 18]*/
+	public int encodeASCII(char[] sa, int sp, byte[] da, int dp, int len) {
```

Comment:
Perhaps it's time to update `decodeASCII()` as well?

Reply:
There is no `StringCoding` `decodeASCII()` counterpart impl, `String.decodeASCII()` was suggested instead https://github.com/eclipse-openj9/openj9/blob/2bb574a4b1c25c041c3aba56f877348538678e36/jcl/src/java.base/share/classes/java/lang/Access.java#L460-L462
It can be done in a separated PR for refactoring.